cgi-bin/DW/BlobStore.pm

```diff
@@ -21,24 +21,33 @@ use v5.10;
 use Log::Log4perl;
 my $log = Log::Log4perl->get_logger( __PACKAGE__ );
 
-use DW::BlobStore::LocalDisk;
-use DW::BlobStore::MogileFS;
-use DW::BlobStore::S3;
+use File::Temp;
+
+LJ::ModuleLoader->require_subclasses('DW::BlobStore');
```

Comment:
Weirdly this was all working last night but I think due to some dark magic... today it wasn't working and this was the fix. Wat. Thanks!